sdk/cargo-build-sbf/src/main.rs

```diff
@@ -105,10 +106,7 @@ where
             writeln!(out, "{key}=\"{value}\" \\").unwrap();
         }
         write!(out, "{}", program.display()).unwrap();
-        for arg in args.iter() {
-            write!(out, " {}", arg.as_ref().to_str().unwrap_or("?")).unwrap();
-        }
-        writeln!(out).unwrap();
+        writeln!(out, "{}", msg).unwrap();
```

Comment:
why this change?
we used to print each arg on a separate line. this will now print tem on the same line separated by spaces, right?

Reply:
where were we getting new lines in the removed code?

Reply:
nevermind I am an idiot. this is fine